tests/config/typo3_rectors.yaml

```diff
@@ -1,13 +1,2 @@
 imports:
-    - { resource: "../../config/typo3-76.yaml" }
-    - { resource: "../../config/typo3-80.yaml" }
-    - { resource: "../../config/typo3-81.yaml" }
-    - { resource: "../../config/typo3-86.yaml" }
-    - { resource: "../../config/typo3-87.yaml" }
-    - { resource: "../../config/typo3-90.yaml" }
-    - { resource: "../../config/typo3-93.yaml" }
-    - { resource: "../../config/typo3-94.yaml" }
-    - { resource: "../../config/typo3-95.yaml" }
-    - { resource: "../../config/typo3-100.yaml" }
-    - { resource: "../../config/typo3-101.yaml" }
-    - { resource: "../../config/typo3-102.yaml" }
+    - { resource: "../../config/*" }
```

Comment:
Symfony glob tip: https://symfony.com/blog/new-in-symfony-3-3-import-config-files-with-glob-patterns